src/Rekono/Provider.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace SocialiteProviders\Rekono;
+
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
```

Comment:
Can we have an `@see` docblock comment with a link to the docs please :)

Reply:
See [comment above](https://github.com/SocialiteProviders/Providers/pull/547#discussion_r506488504)